### ISSUE TITLE
Error Handling: Files Not Found

### ISSUE THREAD

**Issue by bryantdurrell** [dariusk/twitter-archiver#1]
Browser: Safari Mac, Version 16.2 (18614.3.7.1.5)

I was trying to get this to work with a re-created zip file and the tool couldn't find `data/manifest.js`, generating an Unhandled Promise Rejection at line 268. Easy fix on my part, but better error handling would be nice (at a fairly low priority level, since the path I took to get there was low probability). Zip file available on request if you'd find it useful.

Sorry I don't know enough JS to feel confident taking a shot at this!

**Comment by dariusk** [comment]
Thanks Bryant, I'll probably poke at this later this week.

**Comment by dariusk** [comment]
Fixed in 9463c3a

![image](https://user-images.githubusercontent.com/266454/208330702-52a91693-b9d9-4e5b-a63f-ff0a028505c4.png)

**Comment by gobborg** [comment]
I'm getting the error in the above screenshot now.
![image](https://github.com/dariusk/twitter-archiver/assets/81018718/7f23391f-7056-466b-a4d7-360c3b187b3d)

Steps to reproduce:
1. Unzip twitter archive `unzip mytwitterarchive.zip -d mytwitterarchive`
2. Move data/tweets_media/ `mv mytwitterarchive/data/tweets_media/ .`
3. Rezip the archive without the data/tweets_media in it `zip -r mysmallertwitterarchive.zip mytwitterarchive/`

mysmallertwitterarchive.zip is 792 megabytes
```
gobborg@lemur4:~/Downloads$ ls -lh mysmallertwitterarchive.zip 
-rw-r--r--    1 gobborg  gobborg   792.0M Aug  3 08:32 mysmallertwitterarchive.zip
```

Environment info:
Firefox on Linux.

```
gobborg@lemur4:~/Downloads$ apk info firefox
firefox-109.0.1-r0 description:
Firefox web browser

firefox-109.0.1-r0 webpage:
https://www.firefox.com/

firefox-109.0.1-r0 installed size:
216 MiB

gobborg@lemur4:~/Downloads$ uname -a
Linux lemur4 5.15.120-0-lts #1-Alpine SMP Wed, 05 Jul 2023 18:59:57 +0000 x86_64 Linux

gobborg@lemur4:~/Downloads$ apk info zip
zip-3.0-r10 description:
Creates PKZIP-compatible .zip files

zip-3.0-r10 webpage:
http://www.info-zip.org/pub/infozip/Zip.html

zip-3.0-r10 installed size:
412 KiB

gobborg@lemur4:~/Downloads$ apk info unzip
unzip-6.0-r13 description:
Extract PKZIP-compatible .zip files

unzip-6.0-r13 webpage:
http://www.info-zip.org/UnZip.html

unzip-6.0-r13 installed size:
320 KiB
```
